packages/modal/src/Modal.tsx

```diff
@@ -200,6 +200,7 @@ function Modal({
     <Transition in={open} timeout={500} mountOnEnter unmountOnExit>
       {(state: string) => (
         <Portal>
+          {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,  jsx-a11y/no-static-element-interactions */}
```

Comment:
Can you explain what this is for / leave a comment in the file? I think as a general standard, any time we disable a linting rule, especially accessibility-related ones, we should have a comment explaining why.

Reply:
Was only disabling for now, and was planning on making separate PRs to each component to make them comply with new linter rules. Just wanted to break it up, so it wasn't one large PR. Let me know if you'd prefer if I just make the updates right now, and can just include multiple changesets in this PR.